models/build.js

```diff
@@ -179,6 +179,8 @@ module.exports = {
      */
     create: Joi.object(mutate(MODEL, [
         'jobId'
+    ], [
+        'meta'
```

Comment:
I think we're still using `POST /builds` for build monitoring.

Since the additional work to add it to this endpoint is relatively trivial, I think it should be alright to do so.

Reply:
Until we are clear on who all using it, lets keep both.

Is it easy to replace the logic in POST `/builds` with some internal callback to logic in POST `/events` ?

Reply:
Shouldn't be too bad. Do we want that change to be within the scope of this story?

Reply:
Nope, can be a different effort, this one's good to go.